source/image-handler/test/bitmap-handler/bitmap-handler.spec.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Jimp from 'jimp';
+import { BitmapRequest } from '../../bitmap-handler';
+import fs from 'fs';
+import { ImageHandlerError, StatusCodes } from '../../lib';
+
+// Mock the Jimp module to prevent actual image processing during tests
+jest.mock('jimp', () => ({
+    read: jest.fn()
+}));
+
+describe('BitmapRequest', () => {
```

Comment:
From my point of view, you should not consider it as a `Request`. Please create appropriate tests in
- `source/image-handler/test/image-handler`
- `source/image-handler/test/image-request/infer-image-type.spec.ts`